dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GithubAppCheck.java

```diff
@@ -75,40 +83,70 @@ public GithubAppCheck(final Namespace ns){
      * @return True if github app is installed, false otherwise. 
      */
     protected boolean isGithubAppEnabledOnRepository(String fullRepoName) {
-        Integer maxRetryCount = 5;
-        Boolean isGithubAppInstalled = isGithubAppEnabledOnRepositoryWithGitApi(fullRepoName);
-        if (!isGithubAppInstalled) {
-            for (Integer i = 0; i < maxRetryCount; i++) {
-                isGithubAppInstalled = isGithubAppEnabledOnRepositoryWithGitApi(fullRepoName);
-                if (isGithubAppInstalled) break;
-            }
-        }
-        if (!isGithubAppInstalled) {
-            isGithubAppInstalled = isGithubAppEnabledOnRepositoryWithRenovateApi(fullRepoName);
+        boolean result = isGithubAppEnabledOnRepositoryWithRenovateApi_Retry(fullRepoName);
+        
+        if (!result) {
+            result = isGithubAppEnabledOnRepositoryWithGitApi_Retry(fullRepoName);
         }
```

Comment:
So I think you only want to make this second call if you couldn't get an answer back from the first call.  If you actually got back a response from the first call and the repo isn't onboarded to renovate then I don't think there is a point of making the second call to `isGithubAppEnabledOnRepositoryWithGitApi_Retry`

Reply:
@cjneasbi done